app/experimenter/targeting/constants.py

```diff
@@ -113,6 +113,33 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903_PREFER_REDUCED_MOTION = NimbusTargetingConfig(
+    name=(
+        "First start-up users on Windows 10 1903 (build 18362) or newer, with a "
+        "new profile, needing default, preferring reduced motion"
```

Comment:
Above name is used in drop down and getting longer and can be shortened to         "First start-up users on Windows 10 1903 (build 18362) and no prefer reduced motion"